tests/test_plotting.py

```diff
@@ -0,0 +1,335 @@
+
+        distributions = [dist1, dist2]
+        dependencies = [dep1, dep2]
+
+        mul_dist = MultivariateDistribution(distributions, dependencies)
+
+        # Compute highest density contours with return periods of 1 and 20 years.
+        return_period_1 = 1
+        ts = 1  # Sea state duration in hours.
+        limits = [(0, 20), (0, 20)]  # Limits of the computational domain.
+        deltas = [0.5, 0.5]  # Dimensions of the grid cells.
+        hdc = HighestDensityContour(mul_dist, return_period_1, ts, limits, deltas)
+        c_unsorted = hdc.coordinates
+
+        # Sort the coordinates.
+        c_sorted = sort_points_to_form_continous_line(c_unsorted[0][0],
+                                               c_unsorted[0][1],
+                                               do_search_for_optimal_start=True)
+
+        # Plot the sorted and unsorted contours.
+        fig = plt.figure(figsize=(10, 5), dpi=150)
+        ax1 = fig.add_subplot(121)
+        plot_contour(x=c_unsorted[0][0],
+                     y=c_unsorted[0][1],
+                     ax=ax1,
+                     contour_label=str(return_period_1) + '-yr contour',
+                     line_style='b-')
+        ax1.title.set_text('Unsorted')
+        ax2 = fig.add_subplot(122)
+        plot_contour(x=c_sorted[0],
+                     y=c_sorted[1],
+                     ax=ax2,
+                     contour_label=str(return_period_1) + '-yr contour',
+                     line_style='b-')
+        ax2.title.set_text('Sorted')
+        #plt.show()
+
+
+class PlottingTest(unittest.TestCase):
+
+    def test_plot_contour_without_sample(self):
+        """
+        Plots a contour in the most basic way.
+        """
+
+        # Define dependency tuple.
+        dep1 = (None, None, None)
+        dep2 = (0, None, 0)
+
+        # Define parameters.
+        shape = ConstantParam(1.471)
+        loc = ConstantParam(0.8888)
+        scale = ConstantParam(2.776)
+        par1 = (shape, loc, scale)
+
+        mu = FunctionParam('power3', 0.1000, 1.489, 0.1901)
+        sigma = FunctionParam('exp3', 0.0400, 0.1748, -0.2243)
+
+        # Create distributions.
+        dist1 = WeibullDistribution(*par1)
+        dist2 = LognormalDistribution(mu=mu, sigma=sigma)
+
+        distributions = [dist1, dist2]
+        dependencies = [dep1, dep2]
+
+        mul_dist = MultivariateDistribution(distributions, dependencies)
+
+        test_contour_IForm = IFormContour(mul_dist, 50, 3, 50)
+        contour_hs = test_contour_IForm.coordinates[0][0]
+        contour_tz = test_contour_IForm.coordinates[0][1]
+
+        fig = plt.figure()
+        ax = fig.add_subplot(111)
+
+        plot_contour(contour_hs, contour_tz, ax)
+        #plt.show()
+
+        x_plot, y_plot = ax.lines[0].get_xydata().T
+        self.assertAlmostEqual(y_plot[0], contour_tz[0],
+                               delta=0.001)
+
+    def test_plot_contour_and_sample(self):

```

Comment:
Also the created plot shows no sample points outside of the contour. So the data may not be ideal to test this plot.

Reply:
I agree. It would be better to check whether the plots are correct similar to how I implemented it in def test_plot_contour_without_sample(self):. However, having the tests run through without errors also provides benefits. I would like to keep it as is to proceed making plot.py available in the master branch.